yesod-test/Yesod/Test.hs

```diff
@@ -695,12 +695,15 @@ get url = request $ do
     setUrl url
 
 -- | Follow a redirect, if the last response was a redirect.
+-- | Return Left with an error message if not a redirect
+-- | Return Right with the redirected URL if it was.
```

Comment:
I don't think this is doing what you wanted:

![image](https://cloud.githubusercontent.com/assets/1274145/13934529/eee25056-ef6f-11e5-931c-6183e26c6c0b.png)